graph_shortest_path/routing.py

```diff
@@ -45,16 +50,33 @@ def find_vertex(self, value):
         pass
 
     def bfs(self, start):
+        visited = []
+        vertQueue = Queue()
+        vertQueue.enqueue(start)
+        while vertQueue:
+            currentVert = vertQueue.dequeue()
+        # print(currentVert, end="")
+        for nbr in (self.vertList[currentVert].getConnections()):
+            if nbr not in visited:
+                vertQueue.enqueue(nbr)
+                visited.append(currentVert)
         """
         Breadth-First search from an input starting Vertex
         Should maintain parent references back from neighbors to their parent.
 
         @param {Vertex} start: The starting vertex
         """
         # !!!! IMPLEMENT ME
-        pass
 
     def output_route(self, start):
+        def addEdges(self, f, t, cost=0):
+            if f in self.vertices:
+                if t in self.vertices:
+                    self.vertices[f].addNeighbor(t, cost)
+            else:
+                return "Not present in Graph"
+        else:
+            return "Not present in Graph"
         """
```

Comment:
Output_route just has to do with the printing of the route taken..What is addEdges? Where did it come from?